tests/Mailer/Ownership/RecipientStrategy/SendToCreatorOfOrganisationTest.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Mailer\Ownership\RecipientStrategy;
+
+use CultuurNet\UDB3\Model\ValueObject\Identity\Uuid;
+use CultuurNet\UDB3\Ownership\Repositories\OwnershipItem;
+use CultuurNet\UDB3\ReadModel\DocumentRepository;
+use CultuurNet\UDB3\ReadModel\JsonDocument;
+use CultuurNet\UDB3\User\UserIdentityDetails;
+use CultuurNet\UDB3\User\UserIdentityResolver;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Psr\Log\LoggerInterface;
+
+class SendToCreatorOfOrganisationTest extends TestCase
+{
+    /**
+     * @var UserIdentityResolver&MockObject
+     */
+    private $identityResolver;
+    /**
+     * @var LoggerInterface&MockObject
+     */
+    private $logger;
+    /**
+     * @var DocumentRepository&MockObject
+     */
+    private $organizerRepository;
+    private SendToCreatorOfOrganisation $strategy;
+    private OwnershipItem $ownershipItem;
+    private string $itemId;
+    private string $creator;
+    private string $ownerId;
+
+    protected function setUp(): void
+    {
+        $this->itemId = Uuid::uuid4()->toString();
+        $this->creator = Uuid::uuid4()->toString();
+        $this->ownerId = Uuid::uuid4()->toString();
+        $this->organizerRepository = $this->createMock(DocumentRepository::class);
+
+        $this->ownershipItem = new OwnershipItem(
+            Uuid::uuid4()->toString(),
+            $this->itemId,
+            'organizer',
+            $this->ownerId,
+            'requested'
+        );
+
+        $this->identityResolver = $this->createMock(UserIdentityResolver::class);
+        $this->logger = $this->createMock(LoggerInterface::class);
+        $this->strategy = new SendToCreatorOfOrganisation($this->identityResolver, $this->organizerRepository, $this->logger);
+    }
+
+    /** @test */
+    public function it_gets_owner_details(): void
+    {
+        $ownerDetails = new UserIdentityDetails(
+            $this->ownerId,
```

Comment:
I don't think this is correct. You are testing the `SendToCreatorOfOrganisation` strategy, so I would expect that the creator of the organisation is used?

Reply:
Even more for this Strategy it's very weird you need an `OwnerShipItem` because for the creator no ownership needs to be created. This seems like a code smell and leads to confusing and difficult to maintain code.

Given those extra insights I think your Strategy interface needs the item id as parameter for it's interface and not the ownership item.

Reply:
Renamed tests and now use creator.